Makefile

```diff
@@ -1,10 +1,19 @@
-.PHONY: e2e
-e2e:
+.PHONY: e2e prepare-e2e e2e-scanner e2e-firewall unit-test
+
+e2e: prepare-e2e e2e-scanner e2e-firewall
```

Comment:
For consistency, `e2e` should be either a prefix or a suffix, but not both.

Reply:
Thanks for this @gvre, I left it as a prefix with https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/56a3b536ba49b734d78b8cbb64242e5b79479668